modules/stochastic_tools/include/state_sim/model/events/StateSimEvFailRate.h

```diff
@@ -0,0 +1,45 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef STATESIMEVFAILRATE_H
+#define STATESIMEVFAILRATE_H
+
+#include "StateSimEvTimeBased.h"
+#include "StateSimEvent.h"
+
+/**
+ * A Failure Rate event is a time based that samples the next time the event occures over a normal distribution using the given perameters
+ */
+class StateSimEvFailRate: public StateSimEvTimeBased
+{
+protected:
+  double _lambda;
```

Comment:
In MOOSE we use `Real` for all doubles. I'm not totally opposed to using double for stand-alone classes but I'd prefer it in most cases unless you are really trying to keep it out of MOOSE, for instance, if you have a good set of unit tests or these classes can be instantiated by themselves.